Description: manpage-has-errors-from-man "macro not defined"
 Delete all occurrences of ".lp" and ".iX" in manpages to get rid of
 this lintian warnings.
Author: Matteo Cypriani <mcy@lm7.fr>
Last-Update: 2010-05-12
--- a/doc/diskd.1
+++ b/doc/diskd.1
@@ -33,7 +33,6 @@
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fR
 .TP
@@ -47,7 +46,6 @@
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fRA pre-made HTML can be found at:
 \&\fR\&\f(CW\(ifhttp://www.tux.org/pub/knaff/fdutils\(is\fR
@@ -62,7 +60,6 @@
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fR
 .PP
@@ -70,7 +67,6 @@
 the info version certain examples are difficult to read due to the
 quoting conventions used in info.
 .SH Description
-.iX "p diskd"
 .PP
 The diskd command has the following syntax:
 .PP
@@ -82,7 +78,6 @@
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fR
 .PP
--- a/doc/diskseekd.1
+++ b/doc/diskseekd.1
@@ -33,7 +33,6 @@
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fR
 .TP
@@ -47,7 +46,6 @@
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fRA pre-made HTML can be found at:
 \&\fR\&\f(CW\(ifhttp://www.tux.org/pub/knaff/fdutils\(is\fR
@@ -62,7 +60,6 @@
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fR
 .PP
@@ -70,9 +67,6 @@
 the info version certain examples are difficult to read due to the
 quoting conventions used in info.
 .SH Description
-.iX "p diskseekd"
-.iX "c dust (shaking it off from a drive)"
-.iX "c shaking off dust from a drive"
 .PP
 Several people have noticed that Linux has a bad tendency of killing
 floppy drives. These failures remained completely mysterious, until
@@ -96,7 +90,6 @@
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fR
 .TP
--- a/doc/fdmount.1
+++ b/doc/fdmount.1
@@ -33,7 +33,6 @@
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fR
 .TP
@@ -47,7 +46,6 @@
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fRA pre-made HTML can be found at:
 \&\fR\&\f(CW\(ifhttp://www.tux.org/pub/knaff/fdutils\(is\fR
@@ -62,7 +60,6 @@
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fR
 .PP
@@ -70,11 +67,6 @@
 the info version certain examples are difficult to read due to the
 quoting conventions used in info.
 .SH Description
-.iX "p fdmount"
-.iX "p fdmountd"
-.iX "p fdlist"
-.iX "p fdumount"
-.iX "c automounting"
 .PP
  
 .nf
@@ -97,7 +89,6 @@
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fR
 .PP
--- a/doc/fdrawcmd.1
+++ b/doc/fdrawcmd.1
@@ -33,7 +33,6 @@
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fR
 .TP
@@ -47,7 +46,6 @@
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fRA pre-made HTML can be found at:
 \&\fR\&\f(CW\(ifhttp://www.tux.org/pub/knaff/fdutils\(is\fR
@@ -62,7 +60,6 @@
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fR
 .PP
@@ -70,10 +67,6 @@
 the info version certain examples are difficult to read due to the
 quoting conventions used in info.
 .SH Description
-.iX "p fdrawcmd"
-.iX "c raw command"
-.iX "c low level interaction with floppy driver"
-.iX "c direct interaction with floppy driver"
 .PP
  
 .nf
@@ -85,7 +78,6 @@
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fR
 .PP
@@ -152,7 +144,6 @@
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fR
 .PP
--- a/doc/floppycontrol.1
+++ b/doc/floppycontrol.1
@@ -33,7 +33,6 @@
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fR
 .TP
@@ -47,7 +46,6 @@
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fRA pre-made HTML can be found at:
 \&\fR\&\f(CW\(ifhttp://www.tux.org/pub/knaff/fdutils\(is\fR
@@ -62,7 +60,6 @@
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fR
 .PP
@@ -70,8 +67,6 @@
 the info version certain examples are difficult to read due to the
 quoting conventions used in info.
 .SH Description
-.iX "p floppycontrol"
-.iX "c configuration of floppy driver"
 .PP
  
 .nf
@@ -93,7 +88,6 @@
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fR
 .PP
@@ -114,9 +108,6 @@
 (\fR\&\f(CW\(if/dev/fd0\(is\fR).
 .PP
 .SH One\ time\ actions
-.iX "c ejecting a disk"
-.iX "c flushing floppy cache"
-.iX "c resetting controller"
 .PP
 The following \fR\&\f(CWfloppycontrol\fR options don't set a configuration
 parameter, but perform a one-time action. They are available to anybody
@@ -161,9 +152,6 @@
 this.
 .PP
 .SH Printing\ current\ settings
-.iX "c printing current settings"
-.iX "c determining drive type"
-.iX "c detecting a disk change"
 .IP
 .TP
 \&\fR\&\f(CW-T\fR\ 
@@ -320,9 +308,6 @@
 .RE
 .PP
 .SH Drive\ type\ configuration\ and\ autodetection
-.iX "c CMOS type"
-.iX "c drive type"
-.iX "c swapping drives"
 .PP
 The following options handle the different available drive types, such
 as double density vs. high density vs. extra density drives, and 5 1/4
@@ -392,7 +377,6 @@
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fRmeans to try out the formats whose minor device numbers are 28 (1.44M),
 16 (720KB), 96 (1.76MB), and 100 (1.92MB), in this order. For the 1.76MB
@@ -424,10 +408,6 @@
 create.
 .PP
 .SH Configuration\ of\ the\ disk\ change\ line
-.iX "c disk change line"
-.iX "c Thinkpad"
-.iX "c disk change detection"
-.iX "c disk absent during operation (false alert)"
 .TP
 \&\fR\&\f(CW--broken_dcl\fR\ 
 Assumes that the disk change line of the drive is broken.  If this is
@@ -478,7 +458,6 @@
 Switches off silent disk change line clearing for this drive.
 .PP
 .SH Timing\ Parameters
-.iX "c strange noises during seek"
 .PP
 This section describes how to configure drive timings.  To set these
 parameters, you need superuser privileges. All times are in "jiffy"
--- a/doc/floppymeter.1
+++ b/doc/floppymeter.1
@@ -33,7 +33,6 @@
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fR
 .TP
@@ -47,7 +46,6 @@
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fRA pre-made HTML can be found at:
 \&\fR\&\f(CW\(ifhttp://www.tux.org/pub/knaff/fdutils\(is\fR
@@ -62,7 +60,6 @@
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fR
 .PP
@@ -70,9 +67,6 @@
 the info version certain examples are difficult to read due to the
 quoting conventions used in info.
 .SH Description
-.iX "p floppymeter"
-.iX "c rotation speed"
-.iX "c raw capacity"
 .PP
  
 .nf
@@ -83,7 +77,6 @@
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fR
 .PP
--- a/doc/getfdprm.1
+++ b/doc/getfdprm.1
@@ -33,7 +33,6 @@
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fR
 .TP
@@ -47,7 +46,6 @@
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fRA pre-made HTML can be found at:
 \&\fR\&\f(CW\(ifhttp://www.tux.org/pub/knaff/fdutils\(is\fR
@@ -62,7 +60,6 @@
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fR
 .PP
@@ -70,9 +67,6 @@
 the info version certain examples are difficult to read due to the
 quoting conventions used in info.
 .SH Description
-.iX "p getfdprm"
-.iX "c reading back the geometry information"
-.iX "c geometry information (reading back)"
 .PP
  
 .nf
@@ -82,7 +76,6 @@
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fR
 .PP
--- a/doc/makefloppies.1
+++ b/doc/makefloppies.1
@@ -33,7 +33,6 @@
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fR
 .TP
@@ -47,7 +46,6 @@
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fRA pre-made HTML can be found at:
 \&\fR\&\f(CW\(ifhttp://www.tux.org/pub/knaff/fdutils\(is\fR
@@ -62,7 +60,6 @@
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fR
 .PP
@@ -79,7 +76,6 @@
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fR
 .PP
--- a/doc/setfdprm.1
+++ b/doc/setfdprm.1
@@ -33,7 +33,6 @@
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fR
 .TP
@@ -47,7 +46,6 @@
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fRA pre-made HTML can be found at:
 \&\fR\&\f(CW\(ifhttp://www.tux.org/pub/knaff/fdutils\(is\fR
@@ -62,7 +60,6 @@
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fR
 .PP
@@ -70,9 +67,6 @@
 the info version certain examples are difficult to read due to the
 quoting conventions used in info.
 .SH Description
-.iX "p setfdprm"
-.iX "c setting the geometry information"
-.iX "c geometry information (setting)"
 .PP
  
 .nf
@@ -85,7 +79,6 @@
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fR
 .PP
--- a/doc/superformat.1
+++ b/doc/superformat.1
@@ -33,7 +33,6 @@
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fR
 .TP
@@ -47,7 +46,6 @@
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fRA pre-made HTML can be found at:
 \&\fR\&\f(CW\(ifhttp://www.tux.org/pub/knaff/fdutils\(is\fR
@@ -62,7 +60,6 @@
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fR
 .PP
@@ -70,8 +67,6 @@
 the info version certain examples are difficult to read due to the
 quoting conventions used in info.
 .SH Description
-.iX "p superformat"
-.iX "c formatting disks (non XDF)"
 .PP
  
 .nf
@@ -86,7 +81,6 @@
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fR
 .PP
@@ -289,7 +283,6 @@
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fR
 .PP
@@ -310,7 +303,6 @@
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fR
 .PP
@@ -323,7 +315,6 @@
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fR
 .PP
@@ -336,7 +327,6 @@
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fR
 .PP
@@ -349,7 +339,6 @@
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fR
 .PP
@@ -363,7 +352,6 @@
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fR
 .PP
@@ -377,7 +365,6 @@
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fR
 .PP
@@ -392,7 +379,6 @@
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fR
 .PP
--- a/doc/xdfcopy.1
+++ b/doc/xdfcopy.1
@@ -33,7 +33,6 @@
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fR
 .TP
@@ -47,7 +46,6 @@
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fRA pre-made HTML can be found at:
 \&\fR\&\f(CW\(ifhttp://www.tux.org/pub/knaff/fdutils\(is\fR
@@ -62,7 +60,6 @@
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fR
 .PP
@@ -70,9 +67,6 @@
 the info version certain examples are difficult to read due to the
 quoting conventions used in info.
 .SH Description
-.iX "p xdfcopy"
-.iX "c XDF (formatting and copying disks)"
-.iX "c formatting XDF disks"
 .PP
  
 .nf
@@ -83,7 +77,6 @@
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fR
 .PP
@@ -144,7 +137,6 @@
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fR
 .IP
@@ -157,7 +149,6 @@
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fR
 .TP
